ref/general/microshed.adoc

```diff
@@ -9,34 +9,30 @@
 MicroShed Testing is a Java library that helps you write true-to-production integration tests for your microservice applications with the  link:https://openliberty.io/blog/2019/03/27/integration-testing-with-testcontainers.html[Testcontainers] framework.
 link:https://microshed.org/microshed-testing/[MicroShed Testing] framework analyzes your application from outside the link:https://www.docker.com/why-docker[Docker] container without accessing the application internals.
 
-Integration tests assess microservices as a group, as compared to unit tests, which test each microservice as an individual unit.
-Thus, integration tests are more comprehensive than unit tests.
-However, integration tests can be more complicated and take longer to run.
+Integration tests are more comprehensive than unit tests, however, integration tests can be more complicated and take longer to run.
```

Comment:
you might split this into two sentences

`...than unit tests. However, integration...`